apps/landing/styles.css

```
@@ -2,56 +2,78 @@
+  --accent: #e30613;
+  /* muted status inks (kept restrained, plan-sheet not candy) */
+  --warn: #a86b0d;
+  --ok: #1f7a4d;
+  --rust: #a8552a;
+  /* always-dark surfaces (CTA card, core layer) */
+  --invert-surface: #111110;
+  --invert-ink: #fcfcfa;
+  /* type */
+  --sans: "Archivo", ui-sans-serif, system-ui, -apple-system, "Segoe UI", sans-serif;
+  --serif: "Archivo", ui-sans-serif, system-ui, sans-serif; /* no serif in PLANKOPF */
+  --mono: "Fragment Mono", ui-monospace, "SFMono-Regular", Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)